### PR TITLE
Various fixes to `podman` build.

### PR DESCRIPTION
**Pull request by richardkiss**

### CODE REVIEW

File: .github/workflows/build-test.yml [hunk 1]
```diff
@@ -81,11 +81,12 @@ jobs:
             else export SCND_VERSION="$PY_VERSION"; fi && \
             echo "Exporting path /opt/python/cp$PY_VERSION-cp$SCND_VERSION/bin" && \
             export PATH=/opt/python/cp$PY_VERSION-cp$SCND_VERSION/bin/:$PATH && \
-            /opt/python/cp38-cp38/bin/python -m venv venv && \
+            /opt/python/cp$PY_VERSION-cp$SCND_VERSION/bin/python -m venv venv && \
```
Comment (hoffmang9) [inline .github/workflows/build-test.yml:84]: There was a reason I chose to use only 3.8 here but I can't remember what that was right now.

**Review by hoffmang9 — APPROVED** [review]
+1

File: .github/workflows/build-test.yml (hunk 1 above)
Comment (richardkiss) [reply .github/workflows/build-test.yml:84]: Let me undo this change, I don't believe it's necessary anymore.

Comment (richardkiss) [reply .github/workflows/build-test.yml]: It worked, I'll squash & merge.

**Pull request merged**